monai/apps/pathology/transforms/spatial/dictionary.py

```diff
@@ -79,6 +81,8 @@ class TileOnGridd(Randomizable, MapTransform):
 
     """
 
+    backend = TileOnGrid.backend
```

Comment:
Have you added the backends for `TileOnGrid` transform?

Thanks.

Reply:
I removed it. I will add this in #3379